Pipfile

```diff
@@ -65,6 +65,7 @@ flower = "*"
 kombu = "==4.3"
 singledispatch = "==3.4.0.3"
 backports-abc = "==0.5"
+whitenoise = "*"
```

Comment:
Should this be a `dev-package`?

Reply:
Good question. For this PR, I think the answer would be yes, but I've also been working on an offline "Desktop mode" for Studio, and I'm planning to use this for that as well, which means it would be used for deployments, but of course that isn't landed yet so things could change. Leaving it in regular packages seems "safer" as it works both ways, but open to thoughts on if this could lead to issues.

Reply:
My thoughts were simply that the only use I saw was in the profile settings, which extends off of test settings. Given that, I would expect it to align more with development use only, until it's actually utilized in some production code.

Reply:
Either way, I don't think it will harm anything, so feel free to resolve this if you think it's "safer."

Reply:
Okay, I plan on pushing the Studio Desktop changes to a WIP PR shortly, and in that case it will be used in desktop deployments, so I'm going to resolve this!